### PR TITLE
Added tests for Sign-up page

### PR DESCRIPTION
**Pull request by tmhung88**
Please review #689  before this PR

### CODE REVIEW

File: cypress/utils/test-utils.ts [hunk 1]
```diff
@@ -1,3 +1,13 @@
 export enum Page {
   HOME_PAGE = '/how-to'
+}
+
+export const generatedId = (length: number) => {
```
Comment (drydenwilliams) [inline cypress/utils/test-utils.ts]: Maybe we could use something like `uuid` for this?

File: src/pages/Events/Content/EventsList/EventsList.tsx [hunk 2]
```diff
@@ -78,8 +78,8 @@ export class EventsList extends React.Component<any> {
                 </Flex>
               )}
               <Flex justifyContent={'center'} mt={20}>
-                <Link to={'#'}>
-                  <Button variant={'secondary'}>More Events</Button>
+                <Link to={'#'} style={{ visibility: 'hidden' }}>
```
Comment (drydenwilliams) [inline src/pages/Events/Content/EventsList/EventsList.tsx]: Link around a button, is this correct? Can't we have a link that is styled like a button?

File: src/pages/Howto/Content/Common/Howto.form.tsx [hunk 3]
```diff
@@ -300,6 +300,7 @@ export class HowtoForm extends React.Component<IProps, IState> {
                                 icon={'delete'}
                                 variant={'tertiary'}
                                 sx={{ position: 'absolute' }}
+                                data-cy={'delete'}
```
Comment (drydenwilliams) [inline src/pages/Howto/Content/Common/Howto.form.tsx]: `data-cy={'delete'}` => `data-cy="delete"`

File: cypress/utils/test-utils.ts (hunk 1 above)
Comment (tmhung88) [reply cypress/utils/test-utils.ts]: uuid could be an overkill or not suitable for this case. The id here is part of username/email for sign-up tests.

File: src/pages/Events/Content/EventsList/EventsList.tsx (hunk 2 above)
Comment (tmhung88) [reply src/pages/Events/Content/EventsList/EventsList.tsx]: The button is not working at the moment. I'd like to hide it. There is a separate issue for it.

**Pull request merged**